tasks/data_handling/data_pre_processing.py

```diff
@@ -0,0 +1,149 @@
+import scanpy as sc
+
+def create_count_matrix(file_path : str = None) -> sc.AnnData:
+    """
+    ### Function that transforms h5-file to a count matrix.
+
+    ---
+    
+    #### Args:
+        - file_path (str): Path to a 10x hdf5 file.
+
+    #### Returns: 
+        - AnnData: Count matrix from h5-file in file_path with unique variables
+
+    ---
+
+    Annotated data matrix, where observations/cells are named by their barcode and variables/genes by gene name. Stores the following information:
+
+    `~anndata.AnnData.X`
+        The data matrix is stored
+    `~anndata.AnnData.obs_names`
+        Cell names
+    `~anndata.AnnData.var_names`
+        Gene names
+    `~anndata.AnnData.var`\ `['gene_ids']`
+        Gene IDs
+    `~anndata.AnnData.var`\ `['feature_types']`
+        Feature types
+    ---
+
+    Written: ronjah@chalmers.se
+    """
+    if file_path == None:
+        raise ValueError('Need a path to h5-file.')
+    adata = sc.read_10x_h5(file_path)
+    adata.var_names_make_unique()
+    return adata
+
+
+def quality_control(adata: sc.AnnData = None) -> None:
```

Comment:
When a function doesn't return anything, there's no need to add  `-> None`

Reply:
I thought it might be valuable for readability, but if you prefer I can remove them.